ArduCopter/mode.cpp

```diff
@@ -641,11 +676,11 @@ void Mode::land_run_horizontal_control()
     }
 
     // this variable will be updated if prec land target is in sight and pilot isn't trying to reposition the vehicle
-    bool doing_precision_landing = false;
+    copter.ap.prec_land_active = false;
```

Comment:
i think we can actually remove "copter.ap.prec_land_active" and instead just use the local variable "doing_precision_landing".